anesthesia-monitor/src/index.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import { render } from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import reduxThunk from 'redux-thunk';
+
+import reducers from './reducers';
+import App from './components/App';
 import './index.css';
 
-ReactDOM.render(
-  <App />,
+
+const createStoreWithMiddleware = applyMiddleware(reduxThunk)(createStore);
+
+let store = createStore(reducers, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
```

Comment:
The way you're creating your store looks a little bit off as well. It looks like if you're trying to use `react-redux-firebase` with `redux-thunk`, you need to add some extra arguments to your store creation.  See the documentation [here](https://github.com/prescottprue/react-redux-firebase#using-with-redux-thunk).